data-exporter/src/main/java/org/reactome/release/Main.java

```diff
@@ -0,0 +1,99 @@
+package org.reactome.release;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.neo4j.driver.v1.*;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.*;
+
+/**
+ * Generates post-release export files for NCBI, UCSC and Europe PMC.
+ * @author jweiser
+ */
+public class Main {
```

Comment:
release-common-lib has a ReleaseStep abstract class that can be extended for other steps. You can see how it's been used in GO Update and ChEBI Update. I'd like to try and keep the code for the various steps of the release process conforming to a common interface. One reasons is that each ReleaseStep could share common functionality. If there is a reason the ReleaseStep was not used for this project, could you explain why not? Are there improvements that could be made to ReleaseStep that would have made it easier to use in this project?

Reply:
For sure, I can have it extend the ReleaseStep class.  I remember we talked about that, but I just wasn't thinking of it while making changes here.

Reply:
Usually, I have one class that extends the ReleaseStep (such as `ChebiUpdateStep`) and the Main method uses that.

Reply:
And if you think that ReleaseStep's functionality could be improved, go ahead and do it! For example, the current ReleaseStep provides functionality to create a MySQL driver from the values in a Properties file. If you're using Neo4j, you could add similar functionality to ReleaseStep.